src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -426,26 +431,28 @@
                                  (coord/unknown-coordinate))))))
 
 (defn- protein-extension
-  [ppos pref palt seq-info]
-  (let [{:keys [alt-prot-seq alt-tx-prot-seq ini-offset]} seq-info
-        [_ ins offset _] (diff-bases pref palt)
-        rest-seq (if (= ppos 1)
-                   (-> alt-tx-prot-seq
-                       (subs 0 ini-offset)
-                       reverse
-                       (#(apply str %)))
-                   (-> alt-tx-prot-seq
-                       (subs (+ ini-offset (count alt-prot-seq)))))
-        ter-site (some-> (string/index-of rest-seq (if (= ppos 1) "M" "*")) inc)]
-    (mut/protein-extension (if (= ppos 1) "Met" "Ter")
-                           (coord/protein-coordinate (if (= ppos 1) 1 (+ ppos offset)))
-                           (mut/->long-amino-acid (if (= ppos 1)
-                                                    (last ins)
-                                                    (or (last ins) (first rest-seq))))
-                           (if (= ppos 1) :upstream :downstream)
-                           (if ter-site
-                             (coord/protein-coordinate ter-site)
-                             (coord/unknown-coordinate)))))
+  [ppos pref palt {:keys [ref-prot-seq alt-prot-seq alt-tx-prot-seq c-ter-adjusted-alt-prot-seq ini-offset]}]
+  (if (and (not= ppos 1)
+           (ter-site-same-pos? ref-prot-seq c-ter-adjusted-alt-prot-seq))
+    (mut/protein-no-effect)
```

Comment:
In this function, I added line 435-437 and moved seq-info destructuring to args .
And I just fixed indent of following lines.